### PR TITLE
Include views in getTables results

### PR DESCRIPTION
**Pull request by webrouse**
Solving: https://keboola.atlassian.net/browse/COM-400
Changes:
- Views are included in `getTables` sync action results.

### CODE REVIEW

File: src/test/resources/getTables/singleViewConfig.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "parameters": {
+    "outputFile": "/code/tables.json",
+    "tables": [
+      {
+        "tableName": "EMP_DETAILS_VIEW",
```
Comment (webrouse) [inline src/test/resources/getTables/singleViewConfig.json:6]: V testovacom datasete uz (toto) view existuje, tak ho netreba vytvarat.

File: src/test/resources/getTables/expectedOnlyTablesResults.json [hunk 2]
```diff
@@ -1 +1 @@
-[{"name":"DEPARTMENTS","tablespaceName":"USERS","schema":"HR","owner":"HR","rowCount":27},{"name":"EMPLOYEES","tablespaceName":"USERS","schema":"HR","owner":"HR","rowCount":107},{"name":"JOBS","tablespaceName":"USERS","schema":"HR","owner":"HR","rowCount":19},{"name":"JOB_HISTORY","tablespaceName":"USERS","schema":"HR","owner":"HR","rowCount":10},{"name":"LOCATIONS","tablespaceName":"USERS","schema":"HR","owner":"HR","rowCount":23},{"name":"REGIONS","tablespaceName":"USERS","schema":"HR","owner":"HR","rowCount":4}]
\ No newline at end of file
+[{"name":"DEPARTMENTS","tablespaceName":"USERS","schema":"HR","owner":"HR","rowCount":27},{"name":"EMPLOYEES","tablespaceName":"USERS","schema":"HR","owner":"HR","rowCount":107},{"name":"EMP_DETAILS_VIEW","schema":"HR","owner":"HR","rowCount":0},{"name":"JOBS","tablespaceName":"USERS","schema":"HR","owner":"HR","rowCount":19},{"name":"JOB_HISTORY","tablespaceName":"USERS","schema":"HR","owner":"HR","rowCount":10},{"name":"LOCATIONS","tablespaceName":"USERS","schema":"HR","owner":"HR","rowCount":23},{"name":"REGIONS","tablespaceName":"USERS","schema":"HR","owner":"HR","rowCount":4}]
```
Comment (webrouse) [inline src/test/resources/getTables/expectedOnlyTablesResults.json:1]: Pribudlo: `{"name":"EMP_DETAILS_VIEW","schema":"HR","owner":"HR","rowCount":0},`

File: src/test/resources/getTables/expectedResults.json [hunk 3]
```diff
@@ -1 +1 @@
...
...inalPosition":2,"primaryKey":false,"uniqueKey":false},{"name":"POSTAL_CODE","sanitizedName":"POSTAL_CODE","type":"VARCHAR2","nullable":true,"length":"12","ordinalPosition":3,"primaryKey":false,"uniqueKey":false},{"name":"CITY","sanitizedName":"CITY","type":"VARCHAR2","nullable":false,"length":"30","ordinalPosition":4,"primaryKey":false,"uniqueKey":false},{"name":"STATE_PROVINCE","sanitizedName":"STATE_PROVINCE","type":"VARCHAR2","nullable":true,"length":"25","ordinalPosition":5,"primaryKey":false,"uniqueKey":false},{"name":"COUNTRY_ID","sanitizedName":"COUNTRY_ID","type":"CHAR","nullable":true,"length":"2","ordinalPosition":6,"primaryKey":false,"uniqueKey":false,"foreignKeyName":"LOC_C_ID_FK","foreignKeyRefTable":"HR","foreignKeyRef":"COUNTRY_C_ID_PK"}]},{"name":"REGIONS","tablespaceName":"USERS","schema":"HR","owner":"HR","rowCount":4,"columns":[{"name":"REGION_ID","sanitizedName":"REGION_ID","type":"NUMBER","nullable":false,"length":null,"ordinalPosition":1,"primaryKey":true,"uniqueKey":false,"primaryKeyName":"REG_ID_PK"},{"name":"REGION_NAME","sanitizedName":"REGION_NAME","type":"VARCHAR2","nullable":true,"length":"25","ordinalPosition":2,"primaryKey":false,"uniqueKey":false}]}]
```
Comment (webrouse) [inline src/test/resources/getTables/expectedResults.json:1]: Pribudlo:
```json
{
  "name":"EMP_DETAILS_VIEW",
  "schema":"HR",
  "owner":"HR",
  "rowCount":0,
  "columns":[
    {
      "name":"EMPLOYEE_ID",
      "sanitizedName":"EMPLOYEE_ID",
      "type":"NUMBER",
      "nullable":false,
      "length":"6,0",
      "ordinalPosition":1,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"JOB_ID",
      "sanitizedName":"JOB_ID",
      "type":"VARCHAR2",
      "nullable":false,
      "length":"10",
      "ordinalPosition":2,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"MANAGER_ID",
      "sanitizedName":"MANAGER_ID",
      "type":"NUMBER",
      "nullable":true,
      "length":"6,0",
      "ordinalPosition":3,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"DEPARTMENT_ID",
      "sanitizedName":"DEPARTMENT_ID",
      "type":"NUMBER",
      "nullable":true,
      "length":"4,0",
      "ordinalPosition":4,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"LOCATION_ID",
      "sanitizedName":"LOCATION_ID",
      "type":"NUMBER",
      "nullable":true,
      "length":"4,0",
      "ordinalPosition":5,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"COUNTRY_ID",
      "sanitizedName":"COUNTRY_ID",
      "type":"CHAR",
      "nullable":true,
      "length":"2",
      "ordinalPosition":6,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"FIRST_NAME",
      "sanitizedName":"FIRST_NAME",
      "type":"VARCHAR2",
      "nullable":true,
      "length":"20",
      "ordinalPosition":7,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"LAST_NAME",
      "sanitizedName":"LAST_NAME",
      "type":"VARCHAR2",
      "nullable":false,
      "length":"25",
      "ordinalPosition":8,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"SALARY",
      "sanitizedName":"SALARY",
      "type":"NUMBER",
      "nullable":true,
      "length":"8,2",
      "ordinalPosition":9,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"COMMISSION_PCT",
      "sanitizedName":"COMMISSION_PCT",
      "type":"NUMBER",
      "nullable":true,
      "length":"2,2",
      "ordinalPosition":10,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"DEPARTMENT_NAME",
      "sanitizedName":"DEPARTMENT_NAME",
      "type":"VARCHAR2",
      "nullable":false,
      "length":"30",
      "ordinalPosition":11,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"JOB_TITLE",
      "sanitizedName":"JOB_TITLE",
      "type":"VARCHAR2",
      "nullable":false,
      "length":"35",
      "ordinalPosition":12,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"CITY",
      "sanitizedName":"CITY",
      "type":"VARCHAR2",
      "nullable":false,
      "length":"30",
      "ordinalPosition":13,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"STATE_PROVINCE",
      "sanitizedName":"STATE_PROVINCE",
      "type":"VARCHAR2",
      "nullable":true,
      "length":"25",
      "ordinalPosition":14,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"COUNTRY_NAME",
      "sanitizedName":"COUNTRY_NAME",
      "type":"VARCHAR2",
      "nullable":true,
      "length":"40",
      "ordinalPosition":15,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"REGION_NAME",
      "sanitizedName":"REGION_NAME",
      "type":"VARCHAR2",
      "nullable":true,
      "length":"25",
      "ordinalPosition":16,
      "primaryKey":false,
      "uniqueKey":false
    }]
}

```

File: src/test/resources/getTables/expectedResultsWithSynonyms.json [hunk 4]
```diff
@@ -1 +1 @@
...
...NCE","sanitizedName":"STATE_PROVINCE","type":"VARCHAR2","nullable":true,"length":"25","ordinalPosition":5,"primaryKey":false,"uniqueKey":false},{"name":"COUNTRY_ID","sanitizedName":"COUNTRY_ID","type":"CHAR","nullable":true,"length":"2","ordinalPosition":6,"primaryKey":false,"uniqueKey":false,"foreignKeyName":"LOC_C_ID_FK","foreignKeyRefTable":"HR","foreignKeyRef":"COUNTRY_C_ID_PK"}]},{"name":"REGIONS","tablespaceName":"USERS","schema":"HR","owner":"HR","rowCount":4,"columns":[{"name":"REGION_ID","sanitizedName":"REGION_ID","type":"NUMBER","nullable":false,"length":null,"ordinalPosition":1,"primaryKey":true,"uniqueKey":false,"primaryKeyName":"REG_ID_PK"},{"name":"REGION_NAME","sanitizedName":"REGION_NAME","type":"VARCHAR2","nullable":true,"length":"25","ordinalPosition":2,"primaryKey":false,"uniqueKey":false}]},{"name":"SYNONYMTEST","schema":"SETUPUSER","owner":"SETUPUSER","rowCount":0,"columns":[{"name":"index","sanitizedName":"index","type":"NUMBER","nullable":true,"length":null,"ordinalPosition":1,"primaryKey":false,"uniqueKey":false},{"name":"NAME","sanitizedName":"NAME","type":"VARCHAR2","nullable":true,"length":"64","ordinalPosition":2,"primaryKey":false,"uniqueKey":false}]}]
```
Comment (webrouse) [inline src/test/resources/getTables/expectedResultsWithSynonyms.json:1]: Pribudlo:
```json
{
  "name":"EMP_DETAILS_VIEW",
  "schema":"HR",
  "owner":"HR",
  "rowCount":0,
  "columns":[
    {
      "name":"EMPLOYEE_ID",
      "sanitizedName":"EMPLOYEE_ID",
      "type":"NUMBER",
      "nullable":false,
      "length":"6,0",
      "ordinalPosition":1,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"JOB_ID",
      "sanitizedName":"JOB_ID",
      "type":"VARCHAR2",
      "nullable":false,
      "length":"10",
      "ordinalPosition":2,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"MANAGER_ID",
      "sanitizedName":"MANAGER_ID",
      "type":"NUMBER",
      "nullable":true,
      "length":"6,0",
      "ordinalPosition":3,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"DEPARTMENT_ID",
      "sanitizedName":"DEPARTMENT_ID",
      "type":"NUMBER",
      "nullable":true,
      "length":"4,0",
      "ordinalPosition":4,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"LOCATION_ID",
      "sanitizedName":"LOCATION_ID",
      "type":"NUMBER",
      "nullable":true,
      "length":"4,0",
      "ordinalPosition":5,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"COUNTRY_ID",
      "sanitizedName":"COUNTRY_ID",
      "type":"CHAR",
      "nullable":true,
      "length":"2",
      "ordinalPosition":6,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"FIRST_NAME",
      "sanitizedName":"FIRST_NAME",
      "type":"VARCHAR2",
      "nullable":true,
      "length":"20",
      "ordinalPosition":7,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"LAST_NAME",
      "sanitizedName":"LAST_NAME",
      "type":"VARCHAR2",
      "nullable":false,
      "length":"25",
      "ordinalPosition":8,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"SALARY",
      "sanitizedName":"SALARY",
      "type":"NUMBER",
      "nullable":true,
      "length":"8,2",
      "ordinalPosition":9,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"COMMISSION_PCT",
      "sanitizedName":"COMMISSION_PCT",
      "type":"NUMBER",
      "nullable":true,
      "length":"2,2",
      "ordinalPosition":10,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"DEPARTMENT_NAME",
      "sanitizedName":"DEPARTMENT_NAME",
      "type":"VARCHAR2",
      "nullable":false,
      "length":"30",
      "ordinalPosition":11,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"JOB_TITLE",
      "sanitizedName":"JOB_TITLE",
      "type":"VARCHAR2",
      "nullable":false,
      "length":"35",
      "ordinalPosition":12,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"CITY",
      "sanitizedName":"CITY",
      "type":"VARCHAR2",
      "nullable":false,
      "length":"30",
      "ordinalPosition":13,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"STATE_PROVINCE",
      "sanitizedName":"STATE_PROVINCE",
      "type":"VARCHAR2",
      "nullable":true,
      "length":"25",
      "ordinalPosition":14,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"COUNTRY_NAME",
      "sanitizedName":"COUNTRY_NAME",
      "type":"VARCHAR2",
      "nullable":true,
      "length":"40",
      "ordinalPosition":15,
      "primaryKey":false,
      "uniqueKey":false
    },
    {
      "name":"REGION_NAME",
      "sanitizedName":"REGION_NAME",
      "type":"VARCHAR2",
      "nullable":true,
      "length":"25",
      "ordinalPosition":16,
      "primaryKey":false,
      "uniqueKey":false
    }]
}

```

**Review by ondrajodas — APPROVED** [review]
ok

**Pull request merged**